### DISCUSSION TITLE
im going to try making multiplayer

### DISCUSSION THREAD

**Discussion by nekoify** [aeolus-1/triangleGame.github.io#55] [General]
just gotta set up a websocket server

**Comment by serifonium** [comment]
frontite

**Reply by nekoify** [reply]
yes

**Comment by nekoify** [comment]
so far ive set up a server and the client and the server can now connect

**Comment by aeolus-1** [comment]
where are you hosting it
github is static pages online

**Reply by aeolus-1** [reply]
unless its local it will need money

**Reply by nekoify** [reply]
im built diff
